stories/tab.js

```diff
@@ -0,0 +1,163 @@
+import React from 'react';
+import styles from '@sambego/storybook-styles';
+import { storiesOf } from '@storybook/react';
+import { checkA11y } from 'storybook-addon-a11y';
+import { withInfo } from '@storybook/addon-info';
+import { Box, Island, Counter as UICounter } from '../components';
+import { Store, State } from '@sambego/storybook-state';
+import { action } from '@storybook/addon-actions';
+import { IconTab, TabGroup, TitleTab } from '../components/tab';
+import { baseStyles, centerStyles } from '../.storybook/styles';
+
+const store = new Store({
+  items: [
+    {
+      active: false,
+      href: '#',
```

Comment:
can you change the story to using buttons? When I click a tab, it somehow navigates me to a different page inside storybook which is really annoying.

Reply:
Or use a preventDefault, I don't really care in this case.

Reply:
Yeah I also noticed that, but if I wanted to see the active state I needed to add a href to the anchor tag, right? So once I see it works, I should change it to buttons?

Reply:
Yes, or as an alternative you can use `event.preventDefault();` in the `onClick`'s